app/__init__.py

```diff
@@ -35,6 +43,11 @@ def create_app(config_name):
     return application
 
 
+@login_manager.user_loader
+def load_user(user_id):
```

Comment:
This is true - swithered over this for a while. Argument is any changes to the user should log that user out immediatly. Which wouldn't be possible with a cookie cache. We can optimise this by either:
- adding a timestamp into the session and checking on that expiry - say every 10mins not every request, This logic would be in the load_user method
- basing the user_id of the functionliaty of alternative tokens - problem there is you need to reload the user from the token, so it would need to be two way encryption. But we could use the email and the last password changed date or something.

Was thinking could be it;s own story